addons/web/static/src/webclient/actions/reports/report.scss

```diff
@@ -340,3 +340,57 @@ li.oe-nested {
 hr {
     border: 1px solid
 }
+
+.btn {
+    font-weight: #{$btn-font-weight};
+    line-height: #{$btn-line-height};
+    color: #{$btn-color};
+    background-color: transparent;
+    border: 1px solid transparent;
+    padding: #{$btn-padding-x} #{$btn-padding-y};
+    font-size: #{$btn-font-size};
+    border-radius: 0.25rem;
```

Comment:
Could you explain why this is static? no variable?

Reply:
Because the border-radius defined here is using css variables too so that won't work either https://github.com/odoo/odoo/blob/saas-17.4/addons/web/static/lib/bootstrap/scss/_variables.scss#L849

Reply:
@adr-odoo do you have any news about this ?

Reply:
https://github.com/odoo/odoo/blob/3c63c517395b758b69c70a99f157501d3da0953e/addons/web/static/lib/bootstrap/scss/_root.scss#L103